crates/rolldown/src/stages/link_stage/bind_imports_and_exports.rs

```diff
@@ -470,6 +470,7 @@ impl<'a> BindImportsAndExportsContext<'a> {
     debug_assert!(
       matches!(importee.exports_kind, ExportsKind::Esm | ExportsKind::CommonJs)
         || importee.has_lazy_export
+        || importee.module_type == ModuleType::Empty
```

Comment:
Can't determine module exports kind if the modules type is empty, since the source file is `""`